algorithms/linfa-elasticnet/src/lib.rs

```diff
@@ -65,3 +68,39 @@ impl<F: Float> ElasticNet<F> {
         ElasticNetParams::new().l1_ratio(F::one())
     }
 }
+
+/// MultiTask Elastic Net model
+///
+/// This struct contains the parameters of a fitted multi-task elastic net model. This includes the
+/// coefficients (a 2-dimensional array), (optionally) intercept (a 1-dimensional array), duality gaps
+/// and the number of steps needed in the computation.
+///
+/// ## Model implementation
+///
+/// The block coordinate descent is widely used to solve generalized linear models optimization problems,
+/// like Group Lasso, MultiTask Ridge or MultiTask Lasso. It cycles through a group of parameters and update
+/// the groups separately, holding all the others fixed. The optimization routine stops when a criterion is
+/// satisfied (dual sub-optimality gap or change in coefficients).
+pub struct MultiTaskElasticNet<F> {
```

Comment:
Add the same `serde` derive that's on `ElasticNet`. Also can you add `#[derive(Debug, Clone, PartialEq)]` on both ElasticNet models?